exec/two_player_collision/main.cpp

```diff
@@ -0,0 +1,251 @@
+/*
```

Comment:
Also I'm looking at this again and realizing that actually it's probably simplest to keep this in two separate PRs like we talked about, i.e. one for the "endpoint fix" (which should not involved any new files) and a separate one introducing (and potentially modifying) the new example files. To that end, I'd remove this file (and the example header below) from this PR and move them to the other one which you'll open soon.

Reply:
I removed all the files relevant for the collision example and put them in a new branch. The files I moved are src/two_player_collision_example.cpp, the two_player_collision_example.h file in the include folder, and the exec/two_player_collision folder.

Reply:
I left this in the other PR but it's relevant here, so I'll repeat. Anyway, I think you need to `git rm` the file to remove it from the commit itself. Or maybe it's enough to just `rm` the file and then do `git add -A` to make sure git registers the deleted file. This is causing your merge conflict in the other PR.